docs/02-app/02-api-reference/01-components/image.mdx

```diff
@@ -620,9 +620,11 @@ module.exports = {
 
 ### `dangerouslyAllowSVG`
 
-The default [loader](#loader) does not optimize SVG images for a few reasons. First, SVG is a vector format meaning it can be resized losslessly. Second, SVG has many of the same features as HTML/CSS, which can lead to vulnerabilities without a proper [Content Security Policy](/docs/app/building-your-application/configuring/content-security-policy).
+The default [loader](#loader) does not optimize SVG images for a few reasons. First, SVG is a vector format meaning it can be resized losslessly. Second, SVG has many of the same features as HTML/CSS, which can lead to vulnerabilities without proper [Content Security Policy (CSP) headers](/docs/app/api-reference/next-config-js/headers).
 
-If you need to serve SVG images with the default Image Optimization API, you can set `dangerouslyAllowSVG` inside your `next.config.js`:
+Therefore, we recommended using the [`unoptimized`](#unoptimized) prop when the [`src`](#src) prop is known to be SVG. This happens automatically when `src` ends with `".svg"`.
+
+However, if you need to serve SVG images with the default Image Optimization API, you can set `dangerouslyAllowSVG` inside your `next.config.js`:
```

Comment:
Should we also mention the `dangerouslyAllowSVG` option in the `image` next.config.js page? https://nextjs.org/docs/app/api-reference/next-config-js/images

It would be good to mention all keys in the `image` options on this page.

Reply:
Sure. I never understood why that page only has one config.

Originally we had config separate from props but I think @leerob or @atcastle consolidated them into a single page.

Do we want to split them out again?

Reply:
Feels like we just need stronger SVG guidance: shouldn't be used with `next/image`?

That removes the config option entirely.

Reply:
There are use cases where you might want svg to be cached and served from the same domain if the svg images are external urls. Although you can probably use `rewrites` for this. 

I think the word "dangerous" in the property name is pretty good guidance already.

Regardless, that doesn't answer @delba's question/suggestion 😅